### ISSUE TITLE
Mustache.js script tag should use custom-template attribute instead of custom-element.

### ISSUE THREAD

**Issue by delputnam** [Automattic/amp-wp#614]
Currently all AMP component scripts that are added use the attribute `custom-element` (https://github.com/Automattic/amp-wp/blob/master/includes/amp-post-template-actions.php#L22).

Example of the way this script is currently included:

```html
<script custom-element="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js" async></script>
```

The script `amp-mustache-0.1.js` is a special case and should use the attribute `custom-template` instead of `custom-element` in order to pass AMP validation. For example:

```html
<script custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js" async></script>
```

For more information, see the amp-mustache rules in the AMP validator specification here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-mustache/0.1/validator-amp-mustache.protoascii#L34

And the amp-mustache component documentation here: https://www.ampproject.org/docs/reference/components/amp-mustache#usage

**Comment by delputnam** [comment]
Note: Once I finish my current project, I'll try to submit a PR for this issue.  I just wanted to note it here so that it isn't forgotten in case I don't get to it for some reason.

**Comment by paulschreiber** [comment]
Here's my attempt at a fix.

**Comment by paulschreiber** [comment]
@mjangda

**Comment by mjangda** [comment]
This was addressed in #693
